From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Mon, 9 Jun 2014 15:16:46 +0200
Subject: [067/249] staging: rtl8723au: rtw_do_join23a(): Use correct error
 return values
Origin: https://git.kernel.org/linus/be82728e9c10542688dfc0810afa38fd47f4a06b

Don't mix true/fail with _SUCCESS/_FAIL as return value

Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723au/core/rtw_ioctl_set.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/rtl8723au/core/rtw_ioctl_set.c b/drivers/staging/rtl8723au/core/rtw_ioctl_set.c
index 7b0d3db..28cfea5 100644
--- a/drivers/staging/rtl8723au/core/rtw_ioctl_set.c
+++ b/drivers/staging/rtl8723au/core/rtw_ioctl_set.c
@@ -109,7 +109,7 @@ int rtw_do_join23a(struct rtw_adapter *padapter)
 						 _drv_err_,
 						 ("***Error =>do_goin: rtw_creat"
 						  "ebss_cmd status FAIL***\n"));
-					ret =  false;
+					ret = _FAIL;
 					goto exit;
 				}
 
